### PR TITLE
fixed footer ui and made it responsive

### PR DESCRIPTION
**Pull request by steven4714**
PR for Issue #93 fixed UI for Desktop and made it responsive for mobile

EDIT: Screenshots for iPhone 6/7/8 and Desktop if you want more screenshots I can add more.

Bildschirmfoto 2022-10-15 um 23 32 38

Bildschirmfoto 2022-10-15 um 23 33 02

### CODE REVIEW

**Comment by samipak458** [comment]
Kindly add screen shots of your changes @steven4714

**Comment by samipak458** [comment]
@steven4714 do something spacing in download else is good 👍

**Comment by steven4714** [comment]
Bildschirmfoto 2022-10-17 um 21 31 45
Bildschirmfoto 2022-10-17 um 21 31 54

**Comment by samipak458** [comment]
There is conflict in style.css 
Kindly resolve it

File: style.css [hunk 1]
```diff
@@ -229,6 +229,58 @@ option {
   top: 20px;
 }
 
+
```
Comment (ghost) [inline style.css:232]: A great feature to add. Glad to learn more about mobile responsive.

**Pull request merged**